package.json

```diff
@@ -156,6 +157,7 @@
     "rewire": "^7.0.0",
     "rosie": "^2.1.0",
     "sass": "^1.67.0",
+    "sharp": "^0.33.5",
```

Comment:
Oh, I almost forgot: once you merge, please update the tech radars to include this new technology. [We have one for contributors and one for implementers ](https://docs.communityhealthtoolkit.org/contribute/tech-radar/)